docs/running_libE.rst

```diff
@@ -291,10 +284,16 @@ For example::
     os.environ["OMP_NUM_THREADS"] = 4
 
 set in your simulation script before the Executor submit command will export the setting to your run.
```

Comment:
Can we format the phrase "Executor submit" in some way?

Reply:
done